bin/cmd.js

```diff
@@ -98,6 +100,13 @@ if (argv.subtitles) {
   MPLAYER_EXEC += ' -sub ' + JSON.stringify(argv.subtitles)
   MPV_EXEC += ' --sub-file=' + JSON.stringify(argv.subtitles)
   OMX_EXEC += ' --subtitles ' + JSON.stringify(argv.subtitles)
+
+  var subtitlesServer = http.createServer(
```

Comment:
Oh, nevermind. Just saw that it's nested in an if-statement already. I misread the diff.